lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -161,6 +366,43 @@ describe('AwsCompileFunctions', () => {
   });
 
   describe('#compileFunctions()', () => {
+    let firstCfTemplate;
+
+    before(() => {
+      return fixtures
+        .extend('function', {})
+        .then(fixturePath => {
+          return runServerless({
+            cwd: fixturePath,
+            cliArgs: ['package'],
+          });
+        })
+        .then(data => {
+          firstCfTemplate = data.cfTemplate;
+        });
+    });
+
+    after(fixtures.cleanup);
+
+    it('should create different version ids when reservedConcurrency is used', () => {
```

Comment:
Let's remove this test (it should not be in scope of this PR) also goal is to have version id not changed when `reservedConcurrency` changes (as it's a lambda global property that applies to all versions and aliases)